docs/assets/css/sidenav.css

```diff
@@ -45,4 +45,73 @@
     height: 25px;
     color: black;
     font-weight: bold;
+}
+
```

Comment:
Originally the new sidenav had different styling from the docs sidenav, but the end result is that both side navs use the original styling. Can you confirm that this new css is still needed? Or is this leftover and needs to be removed?

Reply:
Definitely not needed. Thought I'd deleted it already, as a matter of fact.